x-pack/packages/security-solution/ecs_data_quality_dashboard/impl/data_quality/translations.ts

```diff
@@ -292,3 +292,13 @@ export const WARM_PATTERN_TOOLTIP = ({ indices, pattern }: { indices: number; pa
     defaultMessage:
       '{indices} {indices, plural, =1 {index} other {indices}} matching the {pattern} pattern {indices, plural, =1 {is} other {are}} warm. Warm indices are no longer being updated but are still being queried.',
   });
+
+export const POST_RESULT_ERROR_TITLE = i18n.translate(
+  'securitySolutionPackages.ecsDataQualityDashboard.postResultErrorTitle',
+  { defaultMessage: 'Error saving data quality check' }
+);
+
+export const GET_RESULTS_ERROR_TITLE = i18n.translate(
+  'securitySolutionPackages.ecsDataQualityDashboard.getResultErrorTitle',
+  { defaultMessage: 'Error getting past data quality checks' }
```

Comment:
The problem was that when the [`useStoredPatternRollups`](https://github.com/elastic/kibana/pull/173185/files#diff-88df49726a378a0913e281104a3fa5999a4f6c4e0eac040058b85612a6d161f6R63) hook receives updates on the `patterns` parameter, the hook was aborting the first request and the error catch was not checking that, so it was showing the toast for the self-induced error. Some logs:

<img width="463" alt="requests_logs" src="https://github.com/elastic/kibana/assets/17747913/7080a90c-138c-470f-87dd-a211ee905eb3">

I fixed that using an `ignore` flag instead of relying on the abort signal. 

I could not reproduce the problem with results being cleaned after the error toast, I assume there was some problem with the race condition control, where only the "non-alerts" pattern request (first one) made it to the state. 
Using `ignore` we make sure the first request with incomplete `patterns` is ignored, and only the response of the request that started the latest makes it to the state.

Reply:
Thanks for the thorough testing @andrew-goldstein!